main.go

```diff
@@ -130,6 +132,11 @@ func main() {
 		DeprecationOptions:  deprecationOptions,
 	}
 
+	if err = MoveFDBBinaries(); err != nil {
```

Comment:
I think instead of moving the files in the manager code we should just copy the files to the correct place e.g. `/var/output-files/6.1`, `/var/output-files/6.2`, `/var/output-files/6.3`, etc.